packages/wasm-sdk/build.sh

```diff
@@ -7,6 +7,10 @@
 
 set -ex -o pipefail
 
+# Disable LTO for WebAssembly builds
```

Comment:
Please see packages/wasm-drive-verify/build.sh to check if there is sth useful for wasm-sdk build script, and/or consider refactoring these scripts to use only one wasm build script, to don't repeat same functionalities.